src/components/grid-aware/Covid19InfoBoxBlock/Covid19InfoBoxBlock.js

```diff
@@ -0,0 +1,22 @@
+import PropTypes from "prop-types";
+import React from "react";
+
+import s from "./Covid19InfoBoxBlock.module.css";
+
+const Covid19InfoBoxBlock = ({ title, description }) => {
```

Comment:
Nit: Let's capitalize this as `COVID19InfoBoxBlock`. With camelCase, there different styles for how to handle acronyms, but I have a personal preference for making acronyms all caps. [This also matches Airbnb's style guide](https://github.com/airbnb/javascript/tree/f0df3a8680479ff0b897cd98a1eab6b156899214#naming--Acronyms-and-Initialisms), which we mostly follow and have configured in our linters.

```suggestion
const COVID19InfoBoxBlock = ({ title, description }) => {
```

Reply:
👍 Good to know about caps on acronyms. Changed to COVID19InfoBoxBlock.

Reply:
👍 Add to volunteer page as well!